tests/unit_tests/corsika/test_corsika_output.py

```diff
@@ -717,3 +706,152 @@ def test_event_2D_histogram(corsika_output_instance_set_histograms):
     assert np.size(x_edges) == 6
     assert np.sum(hist) == 2
     assert np.shape(hist) == (5, 5)
+
+
+def test_get_bins_max_dist(corsika_output_instance):
+    # Test when bins and max_dist are None
+    bins, max_dist = corsika_output_instance._get_bins_max_dist()
+    assert bins == 32  # half of the maximum bins
+    assert max_dist == 16  # maximum stop value
+
+    # Test when bins and max_dist are provided
+    bins, max_dist = corsika_output_instance._get_bins_max_dist(bins=5, max_dist=15)
+    assert bins == 5
+    assert max_dist == 15
+
+    # Test when only bins is provided
+    bins, max_dist = corsika_output_instance._get_bins_max_dist(bins=7)
+    assert bins == 7
+    assert max_dist == 16  # maximum stop value
+
+    # Test when only max_dist is provided
+    bins, max_dist = corsika_output_instance._get_bins_max_dist(max_dist=12)
+    assert bins == 32  # half of the maximum bins
+    assert max_dist == 12
+
+
+def test_meta_dict(corsika_output_instance_set_histograms):
+    expected_meta_dict = {
+        "CORSIKA version": corsika_output_instance_set_histograms.version,
+        "simtools version": version.__version__,
+        "Original IACT file": corsika_output_instance_set_histograms.input_file.name,
+        "telescope_indices": list(corsika_output_instance_set_histograms.telescope_indices),
+        "individual_telescopes": corsika_output_instance_set_histograms.individual_telescopes,
+        "msg": "Only lower bin edges are given.",
+    }
+    assert corsika_output_instance_set_histograms._meta_dict == expected_meta_dict
+
+
+def test_dict_1D_distributions(corsika_output_instance_set_histograms):
+    expected_dict_1D_distributions = {
+        "wavelength": {
+            "function": "get_photon_wavelength_distr",
+            "file name": "hist_1D_photon_wavelength_distr",
+            "title": "Wavelength distribution",
+            "edges": "Wavelength",
+            "edges unit":
+                corsika_output_instance_set_histograms.hist_config["hist_position"]["z axis"][
+                    "start"].unit
+        }
+    }
+    assert corsika_output_instance_set_histograms._dict_1D_distributions["wavelength"] == \
```

Comment:
I think we agreed not to use `\` to break lines. If black cannot handle it on it's own, please break it manually with `()` instead (there are more cases like this below).

Reply:
This was done automatically by pycharm. I changed now whenever I saw the slashes. I remember that what we agreed not using slashes to break strings, specifically. Although the parenthesis seem indeed to be the preferred way.